### PR TITLE
Add jenkins & sonar & upgrade collector versions

### PR DESCRIPTION
**Pull request by aochsner**
Migrated to github-graphql-scm-collector 

Have a set of PRs coming to upgrade dependencies and wanted to test it out before initiating the PRs.  Don't have a Jenkins or Sonar instance outside my company's VPN so uplifted this to include those components.

It is configured to build/scan an example app https://github.com/aochsner/hygieia-example-app. Can also fork that over under hygieia if you want

### CODE REVIEW

File: hygieia-starter-kit/Dockerfile [hunk 1]
```diff
@@ -27,18 +78,17 @@ COPY /api/* /hygieia/api/
...
 # For adding certs
-RUN ls -1 /hygieia/cacerts | while read cert; do \
-        openssl x509 -outform der -in /hygieia/cacerts/$cert -out $cert.der; \
-        $JAVA_HOME/bin/keytool -import -alias $cert -keystore $JAVA_HOME/jre/lib/security/cacerts -trustcacerts -file $cert.der -storepass changeit -noprompt; \
-        rm $cert.der; \
-    done
-
-RUN systemctl enable mongod; \
-	systemctl enable hygieiaapp.service; \
+#RUN ls -1 /hygieia/cacerts | while read cert; do \
```
Comment (rvema) [inline hygieia-starter-kit/Dockerfile]: should we control this certs setup using a feature flag?

**Review by rvema — CHANGES_REQUESTED** [review]
please address the review comment

File: hygieia-starter-kit/Dockerfile (hunk 1 above)
Comment (aochsner) [reply hygieia-starter-kit/Dockerfile]: I'm not sure why i commented this out... there should be no harm running this if they don't add any additional certs... i'll uncomment and rerum just to make sure

Comment (aochsner) [reply hygieia-starter-kit/Dockerfile]: Updated...plz review again thanks

**Review by rvema — APPROVED** [review]
lgtm

**Pull request merged**